packages/x-charts/src/PieChart/PieArcLabelPlot.tsx

```diff
@@ -46,23 +45,27 @@ export interface PieArcLabelPlotSlotProps {
 }
 
 export interface PieArcLabelPlotProps
-  extends DefaultizedProps<
-    Pick<
-      DefaultizedPieSeriesType,
-      | 'data'
-      | 'faded'
-      | 'highlighted'
-      | 'innerRadius'
-      | 'outerRadius'
-      | 'cornerRadius'
-      | 'paddingAngle'
-      | 'arcLabel'
-      | 'arcLabelMinAngle'
-      | 'id'
-      | 'highlightScope'
-    >,
-    'outerRadius'
+  extends Pick<
+    DefaultizedPieSeriesType,
+    | 'data'
+    | 'faded'
+    | 'highlighted'
+    | 'cornerRadius'
+    | 'paddingAngle'
+    | 'arcLabel'
+    | 'arcLabelMinAngle'
+    | 'id'
+    | 'highlightScope'
   > {
+  /**
+   * The radius between circle center and the begining of the arc.
+   * @default 0
+   */
+  innerRadius?: number;
```

Comment:
Nitpick: It looks like these types could be moved into a separate type and re-used.